beta/nncf/tensorflow/batchnorm_adaptation.py

```diff
@@ -12,6 +12,7 @@
 """
 
 from nncf.common.batchnorm_adaptation import BatchnormAdaptationAlgorithmImpl
+from beta.nncf.tensorflow.initialization import DataLoaderBNAdaptationRunner
```

Comment:
I suggest refactoring and moving logic from the `DataLoaderBNAdaptationRunner` class to the `TFBatchnormAdaptationAlgorithmImpl` class. The `DataLoaderBNAdaptationRunner` is used only in the `TFBatchnormAdaptationAlgorithmImpl` class. I don't see the reason to create this abstraction.

Reply:
Agree, refactored